profiles/disa_stig-rhel7-baseline/controls/V-71939.rb

```diff
@@ -63,3 +63,7 @@
     end
   end
 end
+
+describe sshd_config do
```

Comment:
I think I recently pushed this test too, so may or may not come up as a merge conflict as a heads up.